.github/workflows/publish-test-results.yml

```diff
@@ -61,6 +61,29 @@ jobs:
             echo "BASE_SHA=$base_sha" >> $GITHUB_ENV
           fi
 
+      - name: Read base name file
+        if: ${{ hashFiles('artifacts/extra/base_name') != '' }}
+        run: |
+          base_name=$(cat artifacts/extra/base_name)
```

Comment:
As mentioned IRL, we might want to add some validation of the base name value here as well.

Reply:
something like `[0-9a-zA-Z._/\-]+` maybe